lib/Lolifiers/Blackout.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Monolol\Lolifiers;
+
+
+use Monolol\Lolifier;
+
+class Blackout implements Lolifier
+{
+    const MESSAGE = ' ... hum ... i forgot the rest of this sentence';
+
+    public function isHandling(array $record)
+    {
+        return true;
+    }
+
+    public function lolify(array $record)
+    {
+        $record['message'] = $this->forgotHalfSentence($record['message']);
+
+        return $record;
+    }
+
+    private function forgotHalfSentence($message)
+    {
+        $nbWords = str_word_count($message);
+
+        if($nbWords > 1)
+        {
+            $posHalfSentence = floor(strlen($message) / 2);
+            $message = substr($message, 0, strpos($message, ' ', $posHalfSentence)) . self::MESSAGE;
```

Comment:
I did some tests. All is ok with multiline messages (tried Tourette and this one).

So I prefer @jvengeon solution because it keeps message "as is" (with new lines if LineFormatter is used with allowInlineLineBreaks option). @Taluu solution retains only first line.